### PR TITLE
How to configure external database during server setup

### PR DESCRIPTION
**Pull request by maximenoel8**
# Description

Describe in server setup section how to configure an external database using mgr-setup

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)

# Links 
https://github.com/SUSE/spacewalk/issues/19052

### CODE REVIEW

**Comment by mcalmer** [comment]
removed "4.3" as we only merged it in master

**Comment by maximenoel8** [comment]
How can I add reviewer to this PR ?  I may not have the rights.

**Comment by keichwa** [comment]
I forgot to mention that we'd appreciate if you'd add a changelog entry at the top of the `.changelog` in the toplevel directory of this repo.  Thanks for your help!

**Review by keichwa — CHANGES_REQUESTED** [review]

**Comment by keichwa** [comment]
@maximenoel8 something went wrong with my review submission yesterday.  Now you should see them. If possible, please check them and commit those that make sense to you.  It's mostly just cosmetic stuff.  Thanks!

**Comment by maximenoel8** [comment]
Thanks for the review. I follow your remarks. Just to be sure, is it understandable that all the section I describe need to be in setup_env.sh ?

File: modules/installation-and-upgrade/pages/server-setup.adoc [hunk 1]
```diff
@@ -182,3 +183,88 @@ tail -f /var/log/rhn/reposync/<CHANNEL_NAME>.log
...
+Currently, configuring an external database is not supported by yast2 setup.
+
+You can configure an external database using the [command]``mgr-setup`` command line tool.
+
+====
+This section guides you through {productname} setup using [command]``mgr-setup``.
+
+.Procedure : Configuring server with external database
+
+First create a [path]``setup_env.sh`` file in the [path]``/root`` directory defining the server setup like this:
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/server-setup.adoc:202]:
```suggestion
. First create a [path]``setup_env.sh`` file in the [path]``/root`` directory defining the server setup.
  The details are explained in the following steps.
```

File: modules/installation-and-upgrade/pages/server-setup.adoc [hunk 2]
```diff
@@ -182,3 +183,88 @@ tail -f /var/log/rhn/reposync/<CHANNEL_NAME>.log
...
+
+
+. Here you define your certificate information and password.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/server-setup.adoc:205]:
```suggestion
. Set the variables defining your certificate information and password.
```

File: modules/installation-and-upgrade/pages/server-setup.adoc [hunk 3]
```diff
@@ -182,3 +183,88 @@ tail -f /var/log/rhn/reposync/<CHANNEL_NAME>.log
...
+The information are the same as <<proc-quickstart-certificate-information-yast>>.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/server-setup.adoc:206]:
```suggestion
The information is the same as explained at <<proc-quickstart-certificate-information-yast>>.
```

File: modules/installation-and-upgrade/pages/server-setup.adoc [hunk 4]
```diff
@@ -182,3 +183,88 @@ tail -f /var/log/rhn/reposync/<CHANNEL_NAME>.log
...
+MANAGER_PASS="spacewalk"
+MANAGER_ADMIN_EMAIL="email@prov.com"
+MANAGER_DB_NAME="susemanager"
+MANAGER_DB_HOST="db hostname"
+MANAGER_DB_PORT="db port"
+MANAGER_DB_PROTOCOL="TCP"
+MANAGER_ENABLE_TFTP="Y"
+----
+
+. Define the specific configuration for the external database.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/server-setup.adoc:234]:
```suggestion
. Set the variables defining the specific configuration for the external database.
```

**Review by keichwa — CHANGES_REQUESTED** [review]

**Comment by keichwa** [comment]
> Thanks for the review. I follow your remarks. Just to be sure, is it understandable that all the section I describe need to be in setup_env.sh ?

Thanks for following up!  I'm now proposing additional tweaks for clarity.  I hope I got it right.  If yes, please commit the suggestions.

**Review by jcayouette — APPROVED** [review]
Other than Karl's requested changes looks great!

**Comment by keichwa** [comment]
> I committed your suggestions. Fill free to merge the documentation once it's accepted on your side !

Thanks for all your help!

**Review by keichwa — APPROVED** [review]

**Pull request merged**